commands/general/help.js

```diff
@@ -32,52 +36,47 @@ class Help extends Command {
           return `\n\`${message.settings.prefix}help ${cat.toLowerCase()}\` | Shows ${cat} commands`;
         }
       }).join("");
-
       embed.setDescription(description)
         .addField("Categories", output);
+
     } else {
-      let n = 0;
-      sorted.forEach(c => {
-        if (c.help.category.toLowerCase() === type.toLowerCase()) {
-          n++;
-        }
-      });
+      if (this.client.commands.has(type) || this.client.aliases.has(type)) {
```

Comment:
A simple if no then return check would be better in this case

Reply:
This is just fixing the currently uploaded version, no need for major changes.

Reply:
This wouldnt be major.